ods_ci/tests/Tests/400__ods_dashboard/416__ods_dashboard_settings_accelerator_profiles.robot

```diff
@@ -0,0 +1,167 @@
+    ...                                    tol_value=${ACC_TOLERATION_VALUE}
+    ...                                    tol_seconds=${ACC_TOLERATION_SECONDS}
+    Accelerator Profile Should Be Displayed In The Grid   ${ACC_DISPLAY_NAME}1
+    Verify Accelerator Profile Values Via CLI   ${ACC_DISPLAY_NAME}1   ${ACC_IDENTIFIER}
+    ...                                         ${ACC_DESCRIPTION}   ${ACC_ENABLED}    tolerations=yes
+    ...                                         tol_operator=${ACC_TOLERATION_OPERATOR}
+    ...                                         tol_effect=${ACC_TOLERATION_EFFECT}
+    ...                                         tol_key=${ACC_TOLERATION_KEY}
+    ...                                         tol_value=${ACC_TOLERATION_VALUE}
+    ...                                         tol_seconds=${ACC_TOLERATION_SECONDS}
+
+Modify An Accelerator Profile Using Accelerator Profiles Administration UI
+    [Documentation]    Modify an Accelerator Profile instance from the Administration UI and verify it's content
+    [Tags]  RHOAIENG-3349
+    ...     Sanity
+    [Setup]  Run Keywords    Create An Accelerator Profile Via CLI    ${ACC_DISPLAY_NAME}2
+    ...    AND
+    ...    Open Dashboard And Navigate to Accelerator Profiles From Settings
+    Edit Accelerator Profile    original_display_name=${ACC2_DISPLAY_NAME}
+    ...                         display_name=${ACC2_DISPLAY_NAME}_modified
+    ...                         identifier=${ACC_IDENTIFIER}_modified
+    ...                         description=${ACC_DESCRIPTION2}_modified    tolerations=yes
+    ...                         tol_operator=${ACC_TOLERATION_OPERATOR}
+    ...                         tol_effect=${ACC_TOLERATION_EFFECT}
+    ...                         tol_key=${ACC_TOLERATION_KEY}
+    ...                         tol_value=${ACC_TOLERATION_VALUE}
+    ...                         tol_seconds=${ACC_TOLERATION_SECONDS}
+    Accelerator Profile Should Be Displayed In The Grid   ${ACC2_DISPLAY_NAME}_modified
+    Verify Accelerator Profile Values Via CLI   ${ACC2_DISPLAY_NAME}_modified   ${ACC_IDENTIFIER}_modified
+    ...                                         ${ACC_DESCRIPTION2}_modified   tolerations=yes
+    ...                                         tol_operator=${ACC_TOLERATION_OPERATOR}
+    ...                                         tol_effect=${ACC_TOLERATION_EFFECT}
+    ...                                         tol_key=${ACC_TOLERATION_KEY}
+    ...                                         tol_value=${ACC_TOLERATION_VALUE}
+    ...                                         tol_seconds=${ACC_TOLERATION_SECONDS}
+
+Delete Tolerations from an Accelerator Profile Using Accelerator Profiles Administration UI
+    [Documentation]    Delete Tolerations from  an Accelerator Profile instance from the Administration UI and
+    ...                verify it's content
+    [Tags]  RHOAIENG-3349
+    ...     Sanity
```

Comment:
It should be also in Tier1

Reply:
Done [110b73a](https://github.com/red-hat-data-services/ods-ci/pull/1209/commits/110b73a6aa09d7ed1e07c3691624829a284a5a3b)